insert_by_period/macros/get_period_boundaries.sql

```diff
@@ -12,7 +12,7 @@
             {{ dateadd('millisecond',
                                 -1,
                                 "nullif('" ~ stop_date ~ "','')::timestamp") }},
-            getdate()
+            {{ dbt_utils.current_timestamp() }}
```

Comment:
In dbt v1.3, this changed to `dbt.current_timestamp()` (note `dbt` not `dbt_utils`), and in dbt utils 1.0 (coming very very soon) `current_timestamp()` is being removed. I recommend not creating a dependency on utils and instead using the one provided in the `dbt` namespace directly. 

Note that this would create a requirement of dbt Core 1.3 for this project, which you can enforce by adding a require-dbt-version block https://docs.getdbt.com/reference/project-configs/require-dbt-version